src/headlessMoneyHash.ts

```diff
@@ -690,7 +733,19 @@ export default class MoneyHashHeadless<TType extends IntentType> {
     windowRef?.close();
   }
 
-  async #renderUrlInRedirect(url: string) {
+  async #renderUrlInRedirect(url: string, options?: RenderOptions) {
+    if (!options) {
+      window.location.href = url;
+      return;
+    }
+
+    const { redirectTarget } = options!;
+
+    if (!redirectTarget || redirectTarget === "_self") {
+      window.location.href = url;
+      return;
+    }
+
```

Comment:
we can just use flag easier for read and use,
instead of `redirectTarget` we can just use `newWindow: true`

`_target` & `_self` in my opinion is harder to use than a flag to switch between two modes